src/css/base.scss

```diff
@@ -17,36 +19,48 @@ nav {
   left: 0;
   background-color: black;
   color: $light-steel-blue;
-  overflow: scroll;
 }
-nav li {
+nav button {
   margin-top: 10px;
-  font-size: 20px;
+  font-size: 120%;
   list-style-type: none;
   width: 100%;
+  background: transparent;
+  color: white;
   text-align: center;
+  outline: none;
+  text-transform: uppercase;
+  border-style: none none dashed none;
 }
-li:hover{
+.nav-footer{
+  position: absolute;
+  bottom:10%
+}
+nav button:hover{
   color: $iris;
   cursor: pointer;
+  border-style: none none inset none;
 }
 .user-info{
-  border: 4px solid green;
   display: flex;
-  justify-content: center;
+  align-items: center;
+  text-align: center;
+  flex-direction: column;
   width: 100%;
+  font-size: 12px;
+  @include vertical-padding(20px)
```

Comment:
Interesting, hadn't thought to do a padding function but I bet it's handy to standardize stuff